central/detection/service/service_impl.go

```diff
@@ -220,10 +226,19 @@ func (s *serviceImpl) enrichAndDetect(ctx context.Context, enrichmentContext enr
 		EnforcementOnly: enrichmentContext.EnforcementOnly,
 	}
 
+	var appliedNetpols *augmentedobjs.NetworkPoliciesApplied
+	appliedNetpols, err = s.getAppliedNetpolsForDeployment(ctx, enrichmentContext, deployment)
+	if err != nil {
+		log.Warnf("Could not determine applied network policies for deployment %s. Continuing with deployment enrichment.", deployment.GetName())
+	}
+
+	log.Debugf("Determined applied network policies for deployment %s: %+v", deployment.GetName(), appliedNetpols)
```

Comment:
This needs to be in an `else` branch, otherwise we will see both log lines: the one from l. 232 and this one together.